README.md

```diff
@@ -0,0 +1,5 @@
+# API Particulier auth
+
+La brique d'autorisation de API Particulier.
+
```

Comment:
It could be valuable to explain how this work.
e.g. 

La brique d'autorisation de API Particulier prend en entrée ....
En sortie, elle ...

Reply:
What do you think about adding testing instructions ?